docs/using-pre-commit.md

```diff
@@ -27,35 +27,43 @@ Note that these hooks do not update the `.ipynb` notebook when you pull. Make su
 
 ## Using Jupytext with the pre-commit package manager
 
-Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file:
-```
+Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file to sync all staged notebooks:
+
+```yaml
 repos:
--   repo: local
+-   repo: https://github.com/mwouts/jupytext
+    rev: #CURRENT_TAG/COMMIT_HASH
     hooks:
     - id: jupytext
-      name: jupytext
-      entry: jupytext --to md
-      files: .ipynb
-      language: python
+      args: [--sync]
 ```
 
-Here is another `.pre-commit-config.yaml` example that uses the --pre-commit mode of Jupytext to convert all `.ipynb` notebooks to `py:light` representation and unstage the `.ipynb` files before committing.
+You can provide almost all command line arguments to Jupytext in pre-commit, for example to produce several kinds of output files:
+
+```yaml
+repos:
+-   repo: https://github.com/mwouts/jupytext
+    rev: #CURRENT_TAG/COMMIT_HASH
+    hooks:
+    - id: jupytext
+      args: [--from, ipynb, --to, py:light, --to, markdown]
```

Comment:
Oh sorry there can only be one value for `--to`

Reply:
Ah my bad for that suggestion.

Reply:
We still have to update the doc - would you like to take https://github.com/mwouts/jupytext/commit/07b323b92da78d24b7f6c0887adc8f9d30607f8f?